app/src/interfaces/file/file.vue

```diff
@@ -142,6 +158,7 @@ import { useI18n } from 'vue-i18n';
 import { ref, computed, toRefs } from 'vue';
 import DrawerCollection from '@/views/private/components/drawer-collection.vue';
 import api, { addTokenToURL } from '@/api';
+import { getRootPath } from '@/utils/get-root-path';
```

Comment:
Let's remove this line. Likely the same merge from main issue here, as line 169 now already has the same import statement, thus failing the CI lint check for this PR: https://github.com/directus/directus/runs/7919553314?check_suite_focus=true#step:3:201